script/1_0_2current_en_script/Teach_Me_Ciel-Sensei/QA_06.txt

```diff
@@ -1,55 +1,55 @@
 <Page1>
-OS. OS. OS.%{n}
+OS. OS. OS.
 Phew.
 <Page2>
-I've been pressing the O and S keys for 20 hours now.%{n}
+I've been pressing the O and S keys for 20 hours now.
 With all this typing, I could have developed an app by now.
 <Page3>
-Phew... nothing like a coffee after work.%{n}
-It even makes me feel like I've actually done%{n}my work.//This is minor, but its more like 'It even makes me feel like I've done my work
+Phew... nothing like a coffee after work.
```

Comment:
a coffee is british english. "nothing like some coffee after work"

Reply:
Guilty XDDD